content/zh/tutorial/04-state.md

```diff
@@ -0,0 +1,186 @@
+---
+title: State
```

Comment:
Nit: super, super minor, but usually the markdown titles should match the first header on the page as it goes to form `document.title`.

If you have a reason for using English titles, no problem (and certainly let me know if this is some issue on our site), else, they probably should be switched to also be translated.